.github/workflows/aws_deploy.yml

```diff
@@ -75,17 +82,30 @@ jobs:
 
       - name: Get timestamp
         id: timestamp
-        run: echo "timestamp=$(date +%Y%m%d%H%M%S)" >> $GITHUB_ENV
+        run: echo "timestamp=$(date +%Y%m%d%H%M%S)" >> $GITHUB_OUTPUT
+
+      - name: Checkout at release tag
+        uses: actions/checkout@v3
+        if: inputs.environment == 'production'
+        with:
+          ref: ${{ inputs.release_tag }}
+
+      - name: Get tags
```

Comment:
I know this step is a bit messy - it could be cleaned up, but I just wanted to get something up for PR first.